charts/partners/mavenir/amf/OWNERS

```diff
@@ -2,6 +2,7 @@ chart:
   name: amf
   shortDescription: unknown
 publicPgpKey: unknown
+providerDelivery: True
 users: []
```

Comment:
There is no users for this, I would wait for this to be fixed before we flag it provider delivery

Reply:
user has been added, please proceed the merge